core/templates/pages/about-page/about-page.component.html

```diff
@@ -13,7 +13,13 @@ <h2 class="oppia-about-title oppia-about-section-title" [innerHTML]="'I18N_ABOUT
       <div class="oppia-about-button-container">
         <button type="submit" (click)="onClickVisitClassroomButton()"
                 class="btn oppia-about-button oppia-about-button-browse"
-                [innerHTML]="'I18N_ACTION_VISIT_CLASSROOM' | translate" >
+                [innerHTML]="'I18N_ACTION_VISIT_WEB_CLASSROOM' | translate" >
```

Comment:
The i18n string that corresponds to this key doesn't contain any HTML. Why do we need to bind it to the `innerHTML` property?

Reply:
Saw a few more instances of this below.

Reply:
Ah, I just used the existing structure for this one. But you're right. Fixed it.